numpy/core/src/multiarray/arraytypes.h

```diff
@@ -28,4 +28,31 @@ small_correlate(const char * d_, npy_intp dstride,
                 npy_intp nk, enum NPY_TYPES ktype,
                 char * out_, npy_intp ostride);
 
+/* FastTake definitions (these used to live on the dtypes) */
+NPY_NO_EXPORT int
+npy_fasttake_1(
+        char *, char *, npy_intp *, npy_intp, npy_intp, npy_intp, npy_intp,
+        NPY_CLIPMODE);
```

Comment:
Where's the definition of this?

Reply:
Opst, that is nonsense I forgot to delete again.